crates/bevy_window/src/window.rs

```diff
@@ -559,10 +560,27 @@ pub struct WindowDescriptor {
     pub canvas: Option<String>,
 }
 
+static DEFAULT_WINDOW_TITLE: Lazy<String> = Lazy::new(|| {
+    std::env::current_exe()
+        .ok()
+        .and_then(|it| Some(format!("{} - bevy", it.file_stem()?.to_string_lossy())))
```

Comment:
I'm actually very cool with "debranding" here / only using "bevy" in cases where the executable name cannot be found. I think we should remove the "- bevy" suffix unless someone has a counterargument.

Reply:
Personally, my argument would be that developers should change the default title; `- bevy` makes it clear that it's a 'temporary/working' title.

Reply:
I guess it does help anchor the window to a "bevy development" context and provides incentive for people to manually set the title. It just also increases the odds of people accidentally leaking the "bevy" context in their game releases, which is something I want to avoid.

Reply:
Yeah having thought about this more, I _really_ think we should just use the binary name by itself to avoid this class of mistake / avoid leaking the bevy context.

Reply:
Using the default is still a risk of bevy context leakage no matter what; std's docs on `current_exe` state that it is not unlikely that it will fail, and whenever it does it would rename to `bevy`.

That is, I'm still not convinced.

Reply:
I just don't see the benefit to including `- bevy`. "Making it clear that its a temporary / working title" also translates to "makes it look more like a temporary / working title if you deploy with it". Why actively make apps "less release ready" when there is a very solid chance that games will be released without setting the title, even with the `- bevy`? If the goal is to make it clear a title is temporary and create incentives for people to change it, should we just go all in and call it `TEMPORARY_BEVY_WINDOW_TITLE_PLEASE_CHANGE`? (clearly we shouldn't, I'm just going to extremes to make a point).